util/csvs.py

```diff
@@ -254,6 +255,78 @@ def challengeDataCsvFilename(csv_dir: str) -> str:
     return os.path.join(csv_dir, f)
 
 
+# ========================================================================
+# predictoor csv
+
+
+@enforce_types
```

Comment:
please add these to predictoor/csvs.py to match up with the modularising task and avoid conflicts. Also, please move predictoor-related tests under util/predictoor/test instead of util/test/predictoor (at least for now, it is what Trent recommended).